docs/AzDO-Example.md

```diff
@@ -0,0 +1,145 @@
+# Continuos Integration and Continuos Deployment using Azure DevOps
```

Comment:
Would be great if you can share the yaml file of the build and release pipelines too.

Reply:
Good idea. I'll do this.